package.json

```diff
@@ -25,8 +25,9 @@
     "publish-release": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish --conventional-graduate --exact || echo 'Error: New release can only be published on main branch' >&2",
     "publish-prerelease": "branch=$(git rev-parse --abbrev-ref HEAD) && [ $branch = main ] && yarn lerna publish prerelease --preid beta --dist-tag next --exact || echo 'Error: New release can only be published on main branch' >&2",
     "test": "jest",
-    "test-cli": "lerna run test-headless --stream",
-    "jest": "jest --watch",
+    "test-cli": "cd acceptance-tests && yarn test-headless --stream",
+    "test-docker": "yarn build-docker && docker container run -it --rm --name=hs-cli-container hs-cli-image yarn test",
+    "build-docker": "docker image build --tag hs-cli-image . && docker image prune -f",
```

Comment:
This command will build the docker image using the Dockerfile. If there are any changes to the files in this repo, this command will generate a new image and prune the old one.